src/constants.ts

```diff
@@ -63,6 +63,18 @@ export const REGISTRANT_ALREADY_EXIST_CODE = "registrant.alreadyExist";
 export const REGISTRANT_ALREADY_EXIST_MESSAGE = "registrant.alreadyExist";
 export const REGISTRANT_ALREADY_EXIST_PARAM = "registrant";
 
+export const INTERNAL_SERVER_ERROR = {
+  message: "Internal Server Error!",
+  code: "internalServerError",
```

Comment:
> Look good however, I think the code should be something else other than just the name.

I think, having the simple error objects like this would help in simplifying the complex codebase.